src/components/QuestionButton/QuestionButton.tsx

```diff
@@ -0,0 +1,30 @@
+import type { FC } from 'react';
```

Comment:
Lets move this to zUI, and instead of making it a button, make it a tooltip hover.